dependencies/vaticle/repositories.bzl

```diff
@@ -20,6 +20,6 @@ load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 def vaticle_dependencies():
```

Comment:
In the new version, the auth token passed to the deploy_npm rule, used to deploy the protocol npm package, is now masked in the rule's output logs.